Adamant/Services/AdamantNotificationService.swift

```diff
@@ -283,6 +347,14 @@ extension AdamantNotificationsService {
         UNUserNotificationCenter.current().removeAllDeliveredNotifications()
         UIApplication.shared.applicationIconBadgeNumber = customBadgeNumber
     }
+    
+    func setValue(for key: String, value: Bool) {
+        securedStore.set(value, for: key)
+    }
+    
+    func getValue<T: Decodable>(for key: String) -> T? {
```

Comment:
This method does not contribute to anything in my opinion

Reply:
What exactly is wrong? We use this method to get and assign configuration values  for inAppSound, inAppVibrate, and inAppToasts:

        inAppSound = getValue(for: StoreKey.notificationsService.inAppSounds) ?? defaultInAppSound
        inAppVibrate = getValue(for: StoreKey.notificationsService.inAppVibrate) ?? defaultInAppVibrate
        inAppToasts = getValue(for: StoreKey.notificationsService.inAppToasts) ?? defaultInAppToasts

Reply:
Calling it directly from the securedStore is only 8 symbols more. So this method doesn't do any job